### PR TITLE
fix input-files, speed up I/O

### LINKED ISSUE #21: does not work on two input fileGrps anymore
```
ocrd-pagetopdf -I OCR-D-OCR,OCR-D-BIN-WAN -O OCR-D-OUT -P textequiv_level word -P multipage merged --overwrite
Traceback (most recent call last): File "/data/ocr-d/ocrd_all/venv/bin/ocrd", line 8, in <module> sys.exit(cli())
 File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/click/core.py", line 1130, in __call__ return self.main(*args, **kwargs) File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/click/core.py", line 1055, in main rv = self.invoke(ctx)
 File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/click/core.py", line 1657, in invoke return _process_result(sub_ctx.command.invoke(sub_ctx)) 
File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/click/core.py", line 1657, in invoke return _process_result(sub_ctx.command.invoke(sub_ctx))
 File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/click/core.py", line 1404, in invoke return ctx.invoke(self.callback, **ctx.params)
 File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/click/core.py", line 760, in invoke return __callback(*args, **kwargs)
 File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/ocrd/cli/bashlib.py", line 115, in bashlib_input_files for input_file in processor.input_files:
 File "/data/ocr-d/ocrd_all/venv/lib/python3.7/site-packages/ocrd/processor/base.py", line 291, in input_files assert len(ret[0]) == 1, 'Use zip_input_files() instead of input_files when processing multiple input fileGrps'
 AssertionError: Use zip_input_files() instead of input_files when processing multiple input fileGrps 13:54:28.798
```

### PR DESCRIPTION
**Pull request by bertsky**
fixes #21

### CODE REVIEW

**Comment by bertsky** [comment]
Note: depends on https://github.com/OCR-D/core/pull/1031 – so in anticipation I required a new ocrd__minversion 2.50.0 here.

File: ocrd-pagetopdf [hunk 1]
```diff
@@ -128,34 +133,37 @@ main () {
...
+        #     options=()
+        # fi
+        # if [[ "${ocrd__argv[overwrite]}" = true ]]; then
+        #     options+=( --force )
+        # fi
+        # options+=( -G $out_file_grp
+        #            -m application/pdf
+        #            -i "$out_id"
+        #            "$out_file" )
+        # ocrd workspace add "${options[@]}"
```
Comment (stweil) [inline ocrd-pagetopdf:165]: So the output file is no longer added to the METS file?

Comment (bertsky) [reply ocrd-pagetopdf:165]: Of course not. See commit msg.

Comment (stweil) [reply ocrd-pagetopdf:165]: Why was that code deactivated instead of simply removed?

Comment (bertsky) [reply ocrd-pagetopdf:165]: Because it still illustrates what the more efficient FIFO-bulk-add loop does.

**Comment by bertsky** [comment]
Actually, I believe we should not allow multiple input fileGrps any longer. The above does serve as an example how this can be done with bashlib, but it is no longer useful for ocrd-pagetopdf – since we decided to have both PAGE annotations and derived images in the same output fileGrp in OCR-D some 2+ years ago. Before that, we could simply pass the (pure) image fileGrp for the derived images to use. But now, that second fileGrp will itself likely contain PAGE files, which only reference the derived images. 

So it makes more sense to use our usual **feature selector/filter** mechanism to decide which images should be used, e.g. `-P feature_selector binarized -P feature_filter clipped,dewarped` (instead of `-I OCR-D-OCR,OCR-D-BIN`).

Of course, that would necessitate extracting (page-level) derived images from the PAGE from bash, but we already use a Python script to extract the original image, so it's not a big deal.

**Comment by bertsky** [comment]
@stweil this bugfix release should be merged ASAP.

As a follow-up, I'll rewrite everything in Python (using feature_filter / feature_selector as described).

**Pull request merged**